tests/situations/commun/vues/qcm.test.js

```diff
@@ -22,25 +22,25 @@ describe('La vue de la question QCM', function () {
   it('affiche des radios', function () {
     question.choix = [1, 2, 3, 4, 5];
     const vue = shallowMount(VueQCM, { localVue, propsData: { question } });
-    expect(vue.findAll('input[type=radio]').length).to.equal(5);
+    expect(vue.findAll('input[type=radio]').length).toBe(5);
   });
 
   it('affiche un control audio sur chaque réponse', function () {
     question.choix = [{ audio: '1' }, { audio: '1' }, { audio: '1' }];
     const vue = shallowMount(VueQCM, { localVue, propsData: { question } });
-    expect(vue.findAll(LecteurAudio).length).to.equal(3);
+    expect(vue.findAllComponents(LecteurAudio).length).toBe(3);
   });
 
   it('affiche une image sur chaque réponse', function () {
     question.choix = [{ image: '1' }, { image: '1' }, { image: '1' }];
     const vue = shallowMount(VueQCM, { localVue, propsData: { question } });
-    expect(vue.findAll('img').length).to.equal(3);
+    expect(vue.findAll('img').length).toBe(3);
   });
 
   describe('affiche une question de type numérique', function () {
     let uneQuestionNumerique;
 
-    before(function () {
+    beforeEach(function () {
```

Comment:
J'aurais tendance à plutôt utiliser `beforeAll` pour rester cohérent avec la version précédente : 
```suggestion
    beforeAll(function () {
```

Reply:
J'ai corrigé en créant plutôt une fonction `uneQuestionNumerique` comme on l'a fait partout ailleurs.